src/lib/minject/InjectorMacro.hx

```diff
@@ -177,15 +177,12 @@ class InjectorMacro
 			if (keep.exists(field.name))
 				field.meta.add(':keep', [], Context.currentPos());
 
-		// sort rtti to ensure post constructors are last and in order
```

Comment:
I didn't think this sort was necessary any more given that we always sort at runtime.